core/modules/widgets/qualify.js

```diff
@@ -40,9 +40,10 @@ QualifyWidget.prototype.execute = function() {
 	// Get our parameters
 	this.qualifyName = this.getAttribute("name");
 	this.qualifyTitle = this.getAttribute("title");
+	this.qualifyTransclusionFootprint = this.getAttribute("footprint") === "yes"
 	// Set context variable
 	if(this.qualifyName) {
-		this.setVariable(this.qualifyName,this.qualifyTitle + "-" + this.getStateQualifier());
+		this.setVariable(this.qualifyName,this.qualifyTitle + "-" + this.getStateQualifier(undefined,this.qualifyTransclusionFootprint));
```

Comment:
IMO you should not rely on the default value used by the `this.getStateQualifier()`. IMO you should name the variable that is used!